reg_tests/xml/vof_resolved.xml

```diff
@@ -0,0 +1,29 @@
+<ParameterList name="MueLu">
+  <Parameter        name="verbosity"                        type="string"   value="none"/>
+  <Parameter        name="coarse: max size"                 type="int"      value="1000"/>
+
+  <Parameter        name="coarse: type"         type="string"   value="RELAXATION"/>
+  <ParameterList    name="coarse: params">
+    <Parameter      name="relaxation: type"     type="string"   value="Symmetric Gauss-Seidel"/>
+    <Parameter      name="relaxation: sweeps"   type="int"      value="30"/>
```

Comment:
I don't understand why this is being done for the coarse solve here. is 30 sweeps of sgs actually faster?

Reply:
That's fine, just curious.